README.md

```diff
@@ -1687,6 +1687,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [Buttons Generator](https://markodenic.com/tools/buttons-generator/) — 100+ buttons you can use in your project.
   * [WrapPixel](https://www.wrappixel.com/) — Download High Quality Free and Premium Admin dashboard template created with Angular, React, VueJs, NextJs and NuxtJs!
   * [Utils.fun](https://utils.fun/en) — All offline daily and development tools based on the browser's computing power, including watermark generation, screen recording, encoding and decoding, encryption and decryption and code formatting, are completely free and do not upload any data to the cloud for processing.
+  * [React Native Starter](https://flatlogic.com/templates/react-native-starter-free) — React Native Starter is a mobile application template that contains tens of ready-to-use components, pages and includes theme support! 
```

Comment:
we list sites that index many templates, those are useful, I dont think we'll list individual templates - feel free to go add it to any of the many templates sites.